README.md

```diff
@@ -1,19 +1,282 @@
-## Node Curriculum Assignment 
 
+# Node Dependency Management
 
-## What the Student Already Knows
+So far, all our code has been written in a single file. But for big projects, writing everything in one giant file becomes hard to manage. In this lesson, we'll cover how to spread code out over several files.
 
-Students have covered the basic structure and syntax of small (single-file) command-line applications. They have not, up until this point, used `require`
+## Objectives
+After this lesson, you'll be able to...
 
-## Learning Goal 
+1. Spread code out over several files instead of writing everything in one file.
+2. Explain different export strategies.
+3. Recognize what parts of a file can be used in other files.
 
-This lesson introduces students to dependency management in Node.js . It should demonstrate the different ways that `require()` works — highlighting some of its quirks and virtues — and it should also introduce them to `npm`
 
-## Learning Objectives- Students should be able to: 
+## Life With Only One File
 
-- Pull in external dependencies for their applications
-- Spread their applications out over several files instead of writing everything in one file
-- Explain different export strategies and how to use each strategy in their application
+Imagine that we only had one file to put all of our code in. As our program got bigger, the file would grow in size. Eventually it could be 1,000 or 10,000 or even 100,000 lines of code! It could work. But it would make it hard to navigate our code.
 
+Pretend that you live in a very big house, but that it only has one big room. Everything just piles up in that one room. Maybe the beds are on one side, and clothes and shoes are next to them. Maybe there's a sink and a refrigerator in the middle of this big room. Just like with our one big file, we could try to keep things organized by grouping them together.
```

Comment:
I really like this analogy -- I wonder if we could switch the order of this paragraph and the preceding one?